src/qibo/gates/abstract.py

```diff
@@ -290,6 +291,11 @@ def matrix(self):
         backend = GlobalBackend()
         return self.asmatrix(backend)
 
+    @property
+    def clifford(self) -> bool:
+        """Returns ``True`` if gate is a Clifford gate, else ``False``."""
+        return self._clifford
```

Comment:
Since this is very simple it could also be just an attribute. And then you directly do `self.clifford = True` in the related gates.

Reply:
Done